bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -489,6 +513,10 @@ bool ReplicaImp::relevantMsgForActiveView(const T *msg) {
 
 template <>
 void ReplicaImp::onMessage<PrePrepareMsg>(PrePrepareMsg *msg) {
+  if (isSeqNumToStopAt(lastExecutedSeqNum)) {
+    LOG_INFO(GL, "Message ignored because we have to stop at super stable checkpoint");
```

Comment:
Same as the other log message. E.g., "Ignore PrePrepare message because ..." or similar.